### ISSUE TITLE
Support GPU passthrough

### ISSUE THREAD

**Issue by maximehyh** [canonical/multipass#2503]
Hello, 

I am trying to deploy a container in `microk8s` in `multipass` on `windows` that will be using a Machine Learning model and hence will need access to the host GPU (Windows). 

I know microk8s allows to have access to the host's GPU (https://microk8s.io/docs/addon-gpu) but as I am on windows microk8s's "host" is multipass and I wonder if multipass (Hyper V in my case) can have access to the actual host GPU (windows). A bit like what WSL is doing with CUDA https://docs.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl.

 I tried to find some documentation but nothing really relevant. 

Have you had some previous use cases in the past and an easy way to make such a setup?

Thanks

**Comment by maximehyh** [comment]
Yes I guessed so. OK thanks I'll try to do it manually with Hyper V.

**Comment by townsend2010** [comment]
Hi @JYlag,

There is currently no way to pass through a GPU to Multipass created virtual machines.

**Comment by godber** [comment]
@townsend2010 does this include multipass on Ubuntu as well?
